src/platform/user/profile/vet360/components/AddressField/AddressEditModal.jsx

```diff
@@ -3,14 +3,14 @@ import { connect } from 'react-redux';
 import pickBy from 'lodash/pickBy';
 import { focusElement } from 'platform/utilities/ui';
 
-import { ADDRESS_POU, ADDRESS_TYPES, FIELD_NAMES, USA } from 'vet360/constants';
-
+import { ADDRESS_POU, FIELD_NAMES, USA } from 'vet360/constants';
 import Vet360EditModal from '../base/Vet360EditModal';
-
 import CopyMailingAddress from 'vet360/containers/CopyMailingAddress';
-
 import ContactInfoForm from '../ContactInfoForm';
 
+const MILITARY_CITY_VALUES = ['APO', 'DPO', 'FPO'];
+const MILITARY_STATE_VALUES = ['AA', 'AE', 'AP'];
```

Comment:
I think we have this data centralized at `platform/forms/address/data`. Example - `src/applications/letters/utils/constants.js`

Reply:
You're absolutely right! Updated 👍